source/system/external/stdatomic.ooc

```diff
@@ -0,0 +1,27 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+include ./atomic
+
+atomic_int: extern cover
```

Comment:
Thanks for the comments @shamanas :+1: 

The lack of operators is "by design", because I felt like this interface would be incomplete without the multiplication and division operators (as these operations would not be atomic), so I decided to not have any of them. It's not like people gonna use atomic integers in arithmetic expressions (if they are gonna use them at all), so I think this raw interface is ok for the specific situations where atomics are needed.

> Any particular reason this is not atomic_int: cover from atomic_int?

Good question :) There is no particular reason, we already have such syntax in `FileUnix` and `stdio` wrapper. It feels nice.

> you could define the extern C functions directly inside your types

thanks, this seems like a reasonable thing to do.